x/nexus/keeper/general_message.go

```diff
@@ -113,12 +105,12 @@ func (k Keeper) SetMessageExecuted(ctx sdk.Context, messageID exported.MessageID
 		return fmt.Errorf("general message %s not found", messageID.String())
```

Comment:
.String() isn't needed when `%s` is used, `fmt` calls it automatically due to the Stringer interface